salt/metalk8s/defaults.yaml

```diff
@@ -51,4 +53,36 @@ kubelet:
     authorization:
       mode: AlwaysAllow
 
+ca:
+  cert:
+    name: /etc/kubernetes/pki/ca.crt
+    signing_private_key: /etc/kubernetes/pki/ca.key
+    CN: kubenetes
+    keyusage: "critical digitalSignature, keyEncipherment, keyCertSign"
+    basicConstraints: "critical CA:true"
+    days_valid: 3650
+  signing_policies:
+    api:
+      - minions: '*'
+      - signing_private_key: /etc/kubernetes/pki/ca.key
+      - signing_cert: /etc/kubernetes/pki/ca.crt
+      - keyusage: "critical digitalSignature, keyEncipherment"
+      - extendedKeyUsage: "serverAuth"
+      - days_valid: 365
+
+kube_api:
+  cert:
+    name: /etc/kubernetes/pki/apiserver.crt
+    public_key: /etc/kubernetes/pki/apiserver.key
+    signing_policy: api
+    CN: kube-api
+    subjectAltName:
+      DNS:
+        - kubernetes
+        - kubernetes.default
+        - kubernetes.default.svc
+        - kubernetes.default.svc.cluster.local
+      IP: []
+  service_ip_range: "10.96.0.0/12"
```

Comment:
This is more global configuration.

Reply:
It's `service_ip_range` for all service ? Oh ok